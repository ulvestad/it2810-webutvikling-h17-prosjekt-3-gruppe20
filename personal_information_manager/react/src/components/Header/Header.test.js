import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './Header';

const jestExpect = global.expect;

it('Header renders without crashing', () => {
  shallow(<Header />);
});

it('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  jestExpect(tree).toMatchSnapshot();
});
