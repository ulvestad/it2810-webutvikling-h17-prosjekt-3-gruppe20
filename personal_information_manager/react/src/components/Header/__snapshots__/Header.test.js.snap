// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="bg-primary text-white"
  id="page-top"
>
  <div
    className="container text-center"
  >
    <h2>
      Welcome to the Personal Information Manager
    </h2>
    <p
      className="lead"
    >
      This page helps to keep your personal information organized in a practical way.
    </p>
  </div>
</header>
`;
